README.md

```diff
@@ -1,24 +1,40 @@
-# tpt-phish-report #
-
-[![GitHub Build Status](https://github.com/cisagov/tpt-phish-report/workflows/build/badge.svg)](https://github.com/cisagov/tpt-phish-report/actions)
-[![CodeQL](https://github.com/cisagov/tpt-phish-report/workflows/CodeQL/badge.svg)](https://github.com/cisagov/tpt-phish-report/actions/workflows/codeql-analysis.yml)
-[![Coverage Status](https://coveralls.io/repos/github/cisagov/tpt-phish-report/badge.svg?branch=develop)](https://coveralls.io/github/cisagov/tpt-phish-report?branch=develop)
-[![Known Vulnerabilities](https://snyk.io/test/github/cisagov/tpt-phish-report/develop/badge.svg)](https://snyk.io/test/github/cisagov/tpt-phish-report)
-
-This is a generic skeleton project that can be used to quickly get a
-new [cisagov](https://github.com/cisagov) Python library GitHub
-project started.  This skeleton project contains [licensing
-information](LICENSE), as well as
-[pre-commit hooks](https://pre-commit.com) and
-[GitHub Actions](https://github.com/features/actions) configurations
-appropriate for a Python library project.
-
-## New Repositories from a Skeleton ##
-
-Please see our [Project Setup guide](https://github.com/cisagov/development-guide/tree/develop/project_setup)
-for step-by-step instructions on how to start a new repository from
-a skeleton. This will save you time and effort when configuring a
-new repository!
+# Technical Payload Test Reports (TPT Reports) #
+
+[![GitHub Build Status](https://github.com/cisagov/tpt-reports/workflows/build/badge.svg)](https://github.com/cisagov/tpt-reports/actions)
+[![CodeQL](https://github.com/cisagov/tpt-reports/workflows/CodeQL/badge.svg)](https://github.com/cisagov/tpt-reports/actions/workflows/codeql-analysis.yml)
+[![Coverage Status](https://coveralls.io/repos/github/cisagov/tpt-reports/badge.svg?branch=develop)](https://coveralls.io/github/cisagov/tpt-reports?branch=develop)
+[![Known Vulnerabilities](https://snyk.io/test/github/cisagov/tpt-reports/develop/badge.svg)](https://snyk.io/test/github/cisagov/tpt-reports)
+
+This package is used to generate and deliver CISA Technical Payload Test (TPT)
+Reports. Reports are delivered locally and include an encrypted PDF attachment.
+The package collects raw data and creates an encrypted PDF.
+
+## Requirements ##
+
+- [Python Environment](CONTRIBUTING.md#creating-the-python-virtual-environment)
+
+## Installation ##
+
+- `git clone https://github.com/cisagov/tpt-phish-report.git`
+- `./setup-env`
+
+```console
+Usage:
+  tpt_phish_report_tool.py SERVICENOW_ID ELECTION_NAME DOMAIN_TESTED JSON_FILE_PATH OUTPUT_DIRECTORY [--log-level=LEVEL]
```

Comment:
```suggestion
  tpt-report SERVICENOW_ID ELECTION_NAME DOMAIN_TESTED JSON_FILE_PATH OUTPUT_DIRECTORY [--log-level=LEVEL]
```

Reply:
Sounds good, this will modified together with tpt_reports.py so that they match and after confirmation that get_json_file and parse_json functions have been added.

Reply:
Since, we only added JSON_FILE_PATH, can you change the suggested change to only have that one in the README?

Reply:
Resolved in [f1f44ea](https://github.com/cisagov/tpt-reports/pull/13/commits/f1f44eab865b5f8cfb8ab1abf6ace129432f2e30). README usage now matches tpt_reports.py